AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsView.swift

```diff
@@ -22,51 +22,114 @@ struct ThresholdsSettingsView: View {
     }
     
     var body: some View {
-        Form {
-            VStack(alignment: .leading, spacing: 16) {
-                Text(Strings.SessionCart.heatmapSettingsTitle)
-                    .foregroundColor(.darkBlue)
-                    .font(Fonts.heavyTitle1)
-                Text(Strings.SessionCart.heatmapSettingsdescription)
-                    .foregroundColor(.aircastingGray)
-                    .font(Fonts.regularHeading2)
+        if #available(iOS 15.0, *) {
+            Form {
+                VStack(alignment: .leading, spacing: 16) {
+                    Text(Strings.SessionCart.heatmapSettingsTitle)
+                        .foregroundColor(.darkBlue)
+                        .font(Fonts.heavyTitle1)
+                    Text(Strings.SessionCart.heatmapSettingsdescription)
+                        .foregroundColor(.aircastingGray)
+                        .font(Fonts.regularHeading2)
+                }
+                .padding()
+                
+                Section {
+                    veryHighTextfield
+                    highTextfield
+                    mediumTextfield
+                    lowTextfield
+                    veryLowTextfield
+                }
```

Comment:
All this chunk of code can be even shorter I think

```
var body: some View {
        if #available(iOS 15.0, *) {
            mainBody
            .toolbar {
                ToolbarItemGroup(placement: .keyboard) {
                    Spacer()
                    Button("Done") { hideKeyboard() }
                }
            }
        } else {
            mainBody
            .onTapGesture { hideKeyboard() }
        }
    }
```

where mainBody is:

```
 var mainBody: some View {
        Form {
            VStack(alignment: .leading, spacing: 16) {
                Text(Strings.SessionCart.heatmapSettingsTitle)
                    .foregroundColor(.darkBlue)
                    .font(Fonts.heavyTitle1)
                Text(Strings.SessionCart.heatmapSettingsdescription)
                    .foregroundColor(.aircastingGray)
                    .font(Fonts.regularHeading2)
            }
            .padding()
            
            Section {
                veryHighTextfield
                highTextfield
                mediumTextfield
                lowTextfield
                veryLowTextfield
            }
            .keyboardType(.numberPad)
            
            VStack {
                Button(action: {
                    thresholdValues = thresholdSettingsViewModel.updateToNewThresholds()
                    presenta....
```

Reply:
Nice catch, will be corrected.